content/docs/examples/wip/tutorial/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Tutorial
```

Comment:
I kind of like the idea of putting the tutorial in the examples section, but maybe it could have a little catchier title like `Istio Step-by-Step`?

Reply:
Good point. I propose the following title: _Introduction to microservices on Kubernetes with Istio_
The tutorial is also about Kubernetes, and about microservices in general. My hope is that it will get ranked by Google search, so when the users will search for "Introduction to microservices", they will arrive to our tutorial.

Reply:
@frankbu Actually, the description says it: _A step-by-step introduction to microservices with Kubernetes and Istio_

Reply:
Two things to consider:
1. If the title is really long, we should have a short linktitle so that the index stays clean: e.g., `linktitle: Introducing Microservices` or `linktitle: Microservices with Istio`
2. Istio docs are supposed to be platform independent, so unless it's a blog we shouldn't really have Kubernetes only docs. That said, maybe it's time to reevaluate and decide that the Istio project is aligned with and an addon for Kubernetes, even though it's architecturally possible to support other platforms.

Reply:
But this is a tutorial, so in my opinion, it can choose to present specific technologies: Node.js, Docker, Kuberenetes. The tutorial teaches Kubernetes features as well.

Reply:
@rcaballeromx Note that this is "Introduction to microservices, on Kubernetes with Istio". The goal is to teach the reader microservices, Kubernetes and Istio together. My goal is to make it the most popular search result on Google for "Introduction to microservices" or "tutorial about microservices".